### PR TITLE
Add kafka2trino integration to camelk

### PR DESCRIPTION
**Pull request by lucferbux**
- [x] Adapt camel k operator to support our integrations
- [x] Add configmap for kafka and trino configs and secrets
- [x] Develop first integration

### CODE REVIEW

File: camelk/base/kafka2trino.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+apiVersion: camel.apache.org/v1
+kind: Integration
+metadata:
+  name: kafka2trino
```
Comment (anishasthana) [inline camelk/base/kafka2trino.yaml:4]: We should use a different name for this. My assumption is that we will have a separate integration for each topic, so our options are:
1. Keep current `kafka2trino.yaml` file, and have a list of integration objects within it.
2. Delete this file, create a separate `integration-<topic_name>` file for each topic.

I _think_ I'm liking the first option better.

**Review by anishasthana — CHANGES_REQUESTED** [review]

File: camelk/base/kafka2trino.yaml (hunk 1 above)
Comment (lucferbux) [reply camelk/base/kafka2trino.yaml]: 👌

**Review by anishasthana — COMMENTED** [review]
/lgtm

**Pull request merged**